lib/twitter_cldr/resources/readme_renderer.rb

```diff
@@ -101,6 +99,10 @@ def slice_hash(hash, keys)
           ret
         end
       end
+
+      def tested_ruby_versions
+        File.read(".github/workflows/unit_tests.yml").match(/ruby-version:\s\[(?<versions>(\d\.\d,?\s?)+)\]/)[:versions]
```

Comment:
Why not parse the YAML here to avoid an obscure regex?

Reply:
I actually did that the first time around (that's the random second commit removing `require "yaml"` 😅 ) but found this was faster.

Reply:
Hmm, faster in terms of performance? We (well, really I) only run this command like three times a year, so it doesn't need to be particularly fast 😄

Reply:
Copy. I'll go back to yaml parsing 👍